services/src/main/java/org/keycloak/authentication/authenticators/access/AllowAccessAuthenticator.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.authentication.authenticators.access;
+
+import org.jboss.logging.Logger;
+import org.keycloak.authentication.AuthenticationFlowContext;
+import org.keycloak.authentication.Authenticator;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.RealmModel;
+import org.keycloak.models.UserModel;
+
+/**
+ * Authenticator will always successfully authenticate.
+ * Useful for example in the conditional flows to be used after satisfying the previous conditions.
+ *
+ * @author <a href="mailto:mabartos@redhat.com">Martin Bartos</a>
+ */
+public class AllowAccessAuthenticator implements Authenticator {
+    private final Logger log = Logger.getLogger(AllowAccessAuthenticator.class);
+
+    @Override
+    public void authenticate(AuthenticationFlowContext context) {
+        log.info("Explicitly allowed access to the resource.");
```

Comment:
Is it possible to change this to "log.trace"?

From what I know, the "info" messages are dedicated to important and non-repeating (and non-user actions) related stuff like for example "Database migration finished successfully" at the server startup etc.

Reply:
Ok